cdap-watchdog/src/main/java/co/cask/cdap/logging/gateway/handlers/LogHandler.java

```diff
@@ -110,16 +109,24 @@ private void doGetLogs(HttpResponder responder, LoggingContext loggingContext,
                          long fromTimeSecsParam, long toTimeSecsParam, boolean escape, String filterStr,
                          @Nullable RunRecord runRecord) {
     try {
-      Filter filter = FilterParser.parse(filterStr);
-      long fromTimeMs = TimeUnit.MILLISECONDS.convert(fromTimeSecsParam, TimeUnit.SECONDS);
-      long toTimeMs = TimeUnit.MILLISECONDS.convert(toTimeSecsParam, TimeUnit.SECONDS);
 
-      if (fromTimeMs < 0 || toTimeMs < 0 || toTimeMs <= fromTimeMs) {
-        responder.sendString(HttpResponseStatus.BAD_REQUEST, "Invalid time range. 'start' and 'stop' should be " +
-          "greater than zero and 'stop' should be greater than 'start'.");
+      if (fromTimeSecsParam < -1 || toTimeSecsParam < 0) {
```

Comment:
Yes, actually default for fromTimeSecsParam is -1 if you don't provide it to the handler.